scripts/class/Hero.js

```diff
@@ -5,6 +5,20 @@ class Hero {
     */
     constructor(level){
         //TODO generate initial hero/random monster with random stats
+        this.name = '';
+        this.experience = 0;
+        this.level = level;
+        this.stats = new Stats();
```

Comment:
Finish `Stats()` the Hero constructor should be fully usable before merge. Decide what the stats will be for a new Hero, and what the stats will be for a monster of a specific level.